packages/massa-web3/test/web3/mockData.ts

```diff
@@ -88,6 +88,7 @@ export const mockNodeStatusInfo: INodeStatus = {
     pos_lock_cycles: 64,
     pos_lookback_cycles: 64,
   },
+  chain_id: BigInt(77658366)
```

Comment:
77658366 this constant in tests and mock is duplicated, better declare it somewhere: `export const BUILDNET_CHAIN_ID = 77658366;`

Reply:
Done